.github/workflows/rolling_batch_integration.yml

```diff
@@ -299,14 +299,34 @@ jobs:
             serve -m test=file:/opt/ml/model/test/
           python3 llm/client.py lmi_dist octocoder
           docker rm -f $(docker ps -aq)
-      - name: Test gpt-neox-20b-bitsandbytes
+      - name: Test speculative-llama-13b
         working-directory: tests/integration
         run: |
           rm -rf models
-          python3 llm/prepare.py lmi_dist gpt-neox-20b-bitsandbytes
+          echo -en "SPECULATIVE=true" > docker_env
```

Comment:
what is this for? This is new to me

Reply:
In the current implementation the patch is applied when this env variable is set. @chen3933 to confirm